db/citations.yml

```diff
@@ -8556,6 +8556,46 @@ nakam;jmchema17:
   wwwemail: ''
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/nakam_jmca17.pdf
   year: '2017'
+nanarong;npj24:
+  ackno: This work was funded by Toyota Research Institute, grant number PO-002332.
+  author:
+    - Tanaporn Na Narong
+    - Zoe N. Zachko
+    - Steven B. Torrisi
+    - Simon J. L. Billinge
+  doi: 10.48550/arXiv.2410.17467
+  entrytype: article
+  facility: ''
+  grant: TRI
+  journal: ''
+  month: ''
+  nb: ''
+  note: submitted to npj Comput. Mater.
+  notes: ''
+  number: ''
+  optannote: ''
+  optnote: ''
+  optwwwlanl: ''
+  pages: ''
+  summary_professional: Interpretable machine learning (ML) provides a 
+    generalizable approach for combining information from multiple heterogeneous 
+    spectra. Random forest models were trained on X-ray absorption near-edge 
+    spectra (XANES), atomic pair distribution functions (PDFs), and both inputs 
+    combined, to extract local atomic environments of transition metal cations 
+    in oxides. Feature importance analysis revealed the most informative regions 
+    in both spectra and how the information is balanced between them. Our approach 
+    enables exploration of information content and can inform experimental design 
+    when choosing between different methods.
+  synopsis: Random forest models trained on XANES, PDF, and both combined 
```

Comment:
This needs to be shorter and just capture what the paper is about for our internal reading lists. I would say "original paper demonstrating use of interpretable ML for combining heterogeneous input modalities of XANES and PDF", something like that.